crates/runtime/src/datafusion/builder.rs

```diff
@@ -188,8 +187,6 @@ impl DataFusionBuilder {
         let ctx = SessionContext::new_with_state(state);
         ctx.add_optimizer_rule(Arc::new(BytesProcessedOptimizerRule::new()));
         ctx.register_udf(embeddings::cosine_distance::CosineDistance::new().into());
-        ctx.register_udf(crate::datafusion::udf::Greatest::new().into());
```

Comment:
DF added support for those UDFs few months ago so we don't need our own implementation (added 12 months ago). We have unit tests for those UDFs and they still pass. 

least udf
https://github.com/apache/datafusion/blob/605ccbd831b5308ea95175c2c2fda2272ef33b15/datafusion/functions/src/core/least.rs#L76

greatest
https://github.com/apache/datafusion/blob/605ccbd831b5308ea95175c2c2fda2272ef33b15/datafusion/functions/src/core/greatest.rs